docs/content/integrations/dbt/reference.mdx

```diff
@@ -159,6 +159,56 @@ In your CI/CD workflows for your Dagster and dbt project:
 
 ---
 
+## Leveraging dbt defer with branch deployments
+
+It is possible to leverage [dbt defer](https://docs.getdbt.com/reference/node-selection/defer) by passing a `state_path` to <PyObject object="DbtProject" module="dagster_dbt" />. This allows you to run a subset of models or tests based on the changes made since the state passed to `state_path`.
+
+In practice, this is most useful when combined with branch deployments in Dagster+, to test changes made in your branches. This can be easily done by updating your Dagster code and your CI/CD files.
+
+First, update your Dagster code to pass a `state_path` to your `DbtProject` object. Also, update the dbt command in your `@dbt_assets` definition to pass the defer args using `get_defer_args`.
```

Comment:
I struggle here. You are asking me to pass a `state_path` but I don't know what `state_path` I should be passing. Where is the state saved? How is it saved? Who is creating it?